packages/api/src/defaultStyleOptions.ts

```diff
@@ -180,7 +180,7 @@ const DEFAULT_OPTIONS: Required<StyleOptions> = {
 
   suggestedActionKeyboardFocusIndicatorBorderColor: '#605E5C',
   suggestedActionKeyboardFocusIndicatorBorderRadius: 0,
-  suggestedActionKeyboardFocusIndicatorBorderStyle: 'solid',
+  suggestedActionKeyboardFocusIndicatorBorderStyle: 'dashed',
```

Comment:
Is this a breaking change?

Reply:
Yes, inevitable due to how focus changed. I will log it in `CHANGELOG.md`.